src/libs/bridge/ReactNativeLauncherMessenger.js

```diff
@@ -41,7 +41,7 @@ export default class ReactNativeLauncherMessenger extends MessengerInterface {
    */
   onMessage(event) {
     const data = JSON.parse(event.nativeEvent.data)
-    this.listener({ data })
+    if (this.listener) this.listener({ data })
```

Comment:
I don't like this kind of code. 

`this.listener` is added during the `addMessageListener`. So we should not be able to pass within the `onMessage` function without calling `addMessageListener` first. 

So it's the argument we pass to the `addMessageListener` that is null. So we should not call the `addMessageListener` if the `listener` is null, no? Where are we calling this method?

Reply:
As I say in this commit message, I only have this problem in scenarios when I watch the flagship app and with hot reload.

`addMessageListener` is called directly by post-me itself

Reply:
You should add this comment to the code directly. 👍